packages/layout/src/components/cards/common/interfaces.ts

```diff
@@ -122,3 +122,7 @@ export const defaultPhoneErrorMessages: InputErrorMessages = {
 	empty: 'Phone number cannot be empty',
 	invalid: 'Enter a telephone number, like 0163 960 598 or +44 7700 900 359',
 };
+
+export interface CardContentProps  {
+	isRssCard?: boolean;
```

Comment:
Better to rename this in terms of the component features. The component should know nothing about the applications that are consuming it.